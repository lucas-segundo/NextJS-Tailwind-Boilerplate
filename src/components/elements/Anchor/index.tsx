const Anchor = () => (
  <section>
    <h1>Anchor</h1>
  </section>
)

export default Anchor
