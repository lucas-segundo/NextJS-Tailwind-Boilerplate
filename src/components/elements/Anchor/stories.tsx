import { Story, Meta } from '@storybook/react'
import Anchor from '.'

export default {
  title: 'Anchor',
  component: Anchor
} as Meta

export const Default: Story = () => <Anchor />
