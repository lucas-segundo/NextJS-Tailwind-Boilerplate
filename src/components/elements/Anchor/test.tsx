import { render, screen } from '@testing-library/react'

import Anchor from '.'

describe('<Anchor />', () => {
  it('should render the heading', () => {
    const { container } = render(<Anchor />)

    expect(screen.getByRole('heading', { name: /Anchor/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
