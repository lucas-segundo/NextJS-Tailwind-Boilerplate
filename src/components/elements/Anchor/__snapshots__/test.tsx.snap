// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Anchor /> should render the heading 1`] = `
<section>
  <h1>
    Anchor
  </h1>
</section>
`;
